src/graph.cc

```diff
@@ -525,9 +525,14 @@ struct matches {
   std::vector<StringPiece>::iterator i_;
 };
 
-bool ImplicitDepLoader::LoadDepFile(Edge* edge, const string& path,
-                                    string* err) {
+bool ImplicitDepLoader::LoadDepFile(Edge* edge, const std::string& path,
+                                    std::string* err) {
```

Comment:
please remove `std::` in both cases to show that this function's signature hasn't changed.

Reply:
done, a lot less diff all over graph.h/cc now